CONFIG.example

```diff
@@ -35,7 +29,7 @@ SPHINX_BUILD = sphinx-build
 
 # List of default targets build when running make
 #
-MAKE_ZPKG = $(PANDA_ROOTFS)/make-zpkg
+MAKE_ZPKG_URL = https://raw.githubusercontent.com/PandABlocks/PandABlocks-rootfs/master/make-zpkg
```

Comment:
The original was correct. We should never depend on an internet connection for build, better to specify that the PandABlocks-rootfs repo should be cloned somewhere and use the script from there